creusot/tests/should_succeed/syntax/13_vec_macro.rs

```diff
@@ -0,0 +1,14 @@
+extern crate creusot_contracts;
+
+use creusot_contracts::{vec, *};
```

Comment:
Sadly, rustc won't let us glob import the vec macro (as it shadows the builtin macro)